packages/editor/src/lib/hooks/useCoarsePointer.ts

```diff
@@ -17,14 +17,28 @@ export function useCoarsePointer() {
 			return
 		}
 		if (window.matchMedia) {
-			const mql = window.matchMedia('(pointer: coarse)')
-			const handler = () => {
+			const mql = window.matchMedia('(any-pointer: coarse)')
```

Comment:
is `any-pointer: coarse` still a necessary change then? can it just be `pointer: coarse` as before?

Reply:
I think it is necessary, it's a better to default to the coarse pointer for multiple inputs imo. Imagine using an ipad with keyboard attached, you have no intention of touching the mouse and go to manipulate a shape, but the right handles aren't exposed for you to rotate it on the first touch, it's only on the second touch that it gets exposed for you. If we expose coarse pointer handles by default and you go to use the track pad, the handles are changed as some as the cursor moves. Does that make sense?

Reply:
So the logic is:
- if any of the pointers are coarse, default to coarse mode
- if in coarse mode and a non-coarse pointer event occurs, exit coarse mode
- if in not-coarse mode and a coarse pointer event occurs, enter coarse mode

?

Reply:
Yes, exactly

Reply:
Gotcha gotcha, thanks for the explanation — so it's to help for that default state, makes sense.

Definitely worth a comment! Can you add that context and/or steve's list as a comment in this file somewhere? It would be helpful for future intrepid explorers spelunking in these parts ;)